IdentityCore/src/cache/crypto/MSIDAssymetricKeyKeychainGenerator.m

```diff
@@ -158,11 +158,9 @@ - (MSIDAssymetricKeyPair *)readKeyPairForAttributes:(MSIDAssymetricKeyLookupAttr
             return nil;
         }
         
-        NSDate *creationDate = [privateKeyDict objectForKey:(__bridge NSDate *)kSecAttrCreationDate];
-        
         MSIDAssymetricKeyPair *keypair = [[MSIDAssymetricKeyPair alloc] initWithPrivateKey:privateKeyRef
                                                                                  publicKey:publicKeyRef
-                                                                              creationDate:creationDate];
+                                                                            privateKeyDict:privateKeyDict];
```

Comment:
nit: should we call it something like "privateKeyAttributes"?

Reply:
When we generate keypair, we do not get back the private key dictionary so this param is set to the privateKey attributes but when you query private Key, we get the private key dictionary back so this param gets set to privateKeyDict. So I think we can keep the name as privateKeyDict. Let me know what you think.

Reply:
I don't have a strong preference.

Reply:
Perfect. So leaving it as is for now. :)